spec/helpers/location_helper_spec.rb

```diff
@@ -0,0 +1,67 @@
+# See deprecated test/helpers/location_helper_test.rb
+
+require "rails_helper"
+
+RSpec.describe LocationHelper, type: :helper do
+  describe "#sanitize_name" do
+    it "sanitizes a name" do
+      name = "California, USA; DROP TABLE USERS; <div></div>"
+      result = LocationHelper.sanitize_name(name)
+      expect(result).to eq("California, USA DROP TABLE USERS divdiv")
+    end
+  end
+
+  describe "#truncate_name" do
+    it "truncates a name" do
+      name = "Chania, Chania Municipality, Chania Regional Unit, Region of Crete, Crete, 73135, Greece"
+      result = LocationHelper.truncate_name(name)
+      expect(result).to eq("Chania, Chania Municipality, Chania Regional ...")
```

Comment:
Should we be smarter about how we truncate this names? We should remove the middle part, no? 
E.g. Split by comma, get first and last part, and then just get from front or end until the limit...

Reply:
great idea 👍 changed to get the first two or last two